Data/Sys/GameSettings/GALE01r0.ini

```diff
@@ -404,3 +404,8 @@ C03F0034 2C0E0001
 EC210032 C0010034
 EC210024 39C00000
 281E0000 00000000
+
+[Gecko_RetroAchievements_Verified]
+$Proper 16:9 Widescreen Support [Dan Salvato, mirrorbender]
+$Properly Display in 4:3 [Dan Salvato, mirrorbender]
+$Normal C Stick Functionality in Singleplayer Modes [Zauron]
```

Comment:
Are you sure you want to allow normal C stick functionality in single player? Unlike the other codes you're allowlisting in this PR, this makes the game easier to play.

Reply:
RetroAchievements does have a patch for this.  Because the C-Stick is so integral to Melee at this point, it's considered a standard part of the gameplay and is core to how most people play the game nowadays.

Reply:
As a Melee player myself, I see both sides of the argument. I'd think keeping original game behavior by default is more in the spirit of Dolphin though.

Reply:
This isn't about defaulting it, this change will let RetroAchievements players use it instead of locking them into original.